Library/Homebrew/language/node.rb

```diff
@@ -22,8 +22,19 @@ def self.std_npm_install_args(libexec)
       # tell npm to not install .brew_home by adding it to the .npmignore file
       # (or creating a new one if no .npmignore file already exists)
       open(".npmignore", "a") { |f| f.write("\n.brew_home\n") }
+      # npm 5.0.0 broke the assumption Homebrew made that npm would always
+      # create and install real directories when passed a file path. We can
+      # restore the old behaviour by packing the path first with "npm pack"
+      # which also helpfully spits out the filename it just created allowing
+      # us to pass that as the final argument.
+
       # npm install args for global style module format installed into libexec
-      ["--verbose", "--global", "--prefix=#{libexec}", "."]
+      %W[
+        --verbose
+        --global
+        --prefix=#{libexec}
+        #{Dir.pwd}/#{Utils.popen_read("npm pack").chomp}
```

Comment:
It used to be possible to use `std_npm_install_args` with an arbitrary package name by doing something like

args = std_npm_install_args(libexec/"vendor") << "foo@2"
system "npm", "install", *args

but now it looks like that won't be possible.

Admittedly, I don't think that behavior is actually used in core currently, but I'm wondering if we should allow an optional second argument with arbitrary package name(s), or a splat argument for any additional parameters, or something else along those lines, so that the node language module doesn't lose the flexibility it currently has. (i.e., if second arg is nil, do the pack magic, otherwise pass the parameters along and don't do pack magic.)